src/stacking-group.tsx

```diff
@@ -14,52 +15,30 @@ type Props = {
     children: React.ReactNode;
 };
 
-const StackingGroup: React.FC<Props> = ({moreItemsStyle, stacked = false, maxItems = Infinity, children}) => {
+const StackingGroup: React.FC<Props> = ({moreItemsStyle, stacked = true, maxItems = Infinity, children}) => {
     const isInverse = useIsInverseVariant();
-    const backgroundColor = isInverse ? vars.colors.brandHigh : vars.colors.brandLow;
-    const textColor = isInverse ? vars.colors.textPrimaryInverse : vars.colors.brand;
-    const border = stacked ? `1px solid ${vars.colors.borderLow}` : 'none';
-
-    const styleCircle = moreItemsStyle.type === 'circle';
-    const space = stacked ? -8 : 8;
     const countChildren = React.Children.count(children);
-    const plusCase = countChildren - maxItems;
-
-    const styleBorderRadius = styleCircle ? vars.borderRadii.avatar : vars.borderRadii.container;
+    const moreItemsCount = countChildren - maxItems;
```

Comment:
```suggestion
    const moreItemsCount = countChildren - maxItems + 1;
```
![image](https://github.com/Telefonica/mistica-web/assets/1849576/58851f30-ad25-470d-b4d7-8e4dc7baab4a)